cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -80,6 +80,7 @@
     cronjob at:'30 6 * * *', do:deploy_dir('bin', 'cron', 'admin_progress')
     cronjob at:'10 7 * * *', do:deploy_dir('bin', 'cron', 'retention_stats')
     cronjob at:'*/1 * * * *', do:deploy_dir('aws', 'ci_build'), notify: 'dev+build@code.org'
+    cronjob at:'@reboot', do:dashboard_dir('bin', 'sqs', 'process_queues_daemon restart')
```

Comment:
nevermind on the `-r`, looks like it uses the first argument as the default command when `-r` isn't provided.